cfme/utils/appliance/__init__.py

```diff
@@ -2480,6 +2480,42 @@ def is_ip_available():
         appliance.name = name or cls._default_name
         return appliance
 
+    def set_resolvable_hostname(self):
+        """Try to lookup the hostname based on self.hostname, which is generally an IP
+
+        If a hostname is resolved for the IP, its used to set the hostname via appliance_cli
+        If an IP is resolved for self.hostname, no action is taken
```

Comment:
please add a note that it is relevant to appliances hosted on openstack.

Reply:
done